pages/glossary/programmatically-hidden.md

```diff
@@ -11,11 +11,12 @@ input_aspects:
 An HTML element is _programmatically hidden_ if either it has a [computed][] CSS property `visibility` whose value is not `visible`; or at least one of the following is true for any of its [inclusive ancestors][] in the [flat tree][]:
 
 - has a [computed][] CSS property `display` of `none`; or
+- has a [computed][] CSS property `content-visibility` of `hidden`; or
```

Comment:
This is wrong, an element with `content-visibility:hidden` is not hidden from the accessibility tree. Its content is, but the element itself isn't. To make this more complicated. Even with `content-visibility:hidden`, child nodes can still be used in the accessible name. For example, not only is the following still a button in the accessibility tree, its accessible name is still "Hello world", even though the text isn't visible.

```html
<button style="content-visibility:hidden">
  Hello world
</button>
```

Reply:
There are various caveats on implementing content-visibility: to avoid screen reader detection:
https://drafts.csswg.org/css-contain/#cv-a11y

And there is an outstanding issue on this:
https://github.com/w3c/csswg-drafts/issues/5857

It may be worth holding off on this until there's consensus on how AT should implement this.